### ISSUE TITLE
animated gif is problematic

### ISSUE THREAD

**Issue by roxken** [MicrosoftDocs/windows-uwp#617]
It's huge, it's not necessary, it has accessibility issues. A guideline page should be a showcase for best practices. Can we lose the gif?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 98e70766-0aac-ef17-92fc-12af68e1e860
* Version Independent ID: bd7b93dc-9817-dc31-c2b6-1a554ffea7e3
* Content: [Writing style - UWP app developer](https://docs.microsoft.com/en-us/windows/uwp/design/style/writing-style#feedback)
* Content Source: [windows-apps-src/design/style/writing-style.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/design/style/writing-style.md)
* Service: **unspecified**
* Product: **windows**
* GitHub Login: @QuinnRadich
* Microsoft Alias: **quradic**

**Comment by QuinnRadich** [comment]
Currently, every page in the Style section has an image of similar size at the top of the page, and I'm not inclined to break from that consistent format.

That being said, I agree that the gif can be a little distracting, and the other pages all have static images. I'm reaching out to our Design team to discuss whether or not we want to convert this to a still image. Thank you.

**Comment by GrantMeStrength** [comment]
Interesting.

**Comment by QuinnRadich** [comment]
Clearing out this old issue.

We reviewed our accessibility guidelines. This is a supplementary graphic that contains no vital information, and is of the same size and general scheme as other top-level images in this section. As might be expected given the age of this issue, we don't currently intend to change it.
